x-pack/plugins/siem/public/components/timeline/helpers.tsx

```diff
@@ -4,43 +4,86 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { isEmpty, isNumber } from 'lodash/fp';
+import { isEmpty, isNumber, get } from 'lodash/fp';
+import memoizeOne from 'memoize-one';
 import { StaticIndexPattern } from 'ui/index_patterns';
 
 import { convertKueryToElasticSearchQuery, escapeQueryValue } from '../../lib/keury';
 
 import { DataProvider, DataProvidersAnd, EXISTS_OPERATOR } from './data_providers/data_provider';
+import { BrowserFields } from '../../containers/source';
 
-const buildQueryMatch = (dataProvider: DataProvider | DataProvidersAnd) =>
+const converTimestampFieldToQuery = (field: string, value: string | number) =>
```

Comment:
`converTimestampFieldToQuery` is missing a `t` in `conver`, but now that we're handling all date fields instead of just timestamps, this could (optionally) reflect the generalized behavior and be named `convertDateFieldToQuery`